### PR TITLE
Add FFTW dependencies for Arch Linux and Oracle Linux

### PR DESCRIPTION
**Pull request by Chris00**

### CODE REVIEW

File: packages/conf-fftw3/conf-fftw3.1/opam [hunk 1]
```diff
@@ -8,12 +8,14 @@ build: [["pkg-config" "fftw3"]]
...
 depexts: [
   ["libfftw3-dev"] {os-family = "debian"}
+  ["fftw"] {os-family = "arch"}
   ["fftw-devel"] {os-distribution = "centos"}
   ["fftw-dev"] {os-distribution = "alpine"}
   ["fftw-devel"] {os-distribution = "fedora"}
   ["fftw-devel"] {os-distribution = "rhel"}
   ["libfftw-devel"] {os-distribution = "mageia"}
   ["fftw3-devel"] {os-family = "suse"}
+  ["fftw-dev"] { os-distribution = "oracle" }
```
Comment (mseri) [inline packages/conf-fftw3/conf-fftw3.1/opam:18]:
```suggestion
  ["fftw-dev"] { os-distribution = "ol" }
```

I think it is ol

Comment (Chris00) [reply packages/conf-fftw3/conf-fftw3.1/opam]: Thanks.  (We will find out soon ! 😀)

File: packages/conf-fftw3/conf-fftw3.1/opam [hunk 2]
```diff
@@ -8,12 +8,14 @@ build: [["pkg-config" "fftw3"]]
...
 depexts: [
   ["libfftw3-dev"] {os-family = "debian"}
+  ["fftw"] {os-family = "arch"}
   ["fftw-devel"] {os-distribution = "centos"}
   ["fftw-dev"] {os-distribution = "alpine"}
   ["fftw-devel"] {os-distribution = "fedora"}
   ["fftw-devel"] {os-distribution = "rhel"}
   ["libfftw-devel"] {os-distribution = "mageia"}
   ["fftw3-devel"] {os-family = "suse"}
+  ["fftw-dev"] { os-distribution = "ol" }
```
Comment (mseri) [inline packages/conf-fftw3/conf-fftw3.1/opam:18]:
```suggestion
  ["fftw-devel"] { os-distribution = "ol" }
```

**Comment by mseri** [comment]
Worked :) thanks!

**Comment by Chris00** [comment]
Thanks for your help!

**Pull request merged**